pkgs/by-name/he/helm-sops/package.nix

```
@@ -0,0 +1,34 @@
+{
+  lib,
+  stdenv,
+  buildGoModule,
+  fetchFromGitHub,
+  helm,
+  sops,
+}:
+
+buildGoModule rec {
+  pname = "helm-sops";
+  version = "20250205-1";
+  src = fetchFromGitHub {
+    owner = "camptocamp";
+    repo = "helm-sops";
+    rev = "20250205-1";
+    sha256 = "sha256-xkHv+PM2acwk9uwAHlVgBbJKhofAU60KJykMEx7Zq8I=";
+  };
+
+  vendorHash = "sha256-jynNpi9XRaLLW1rbvFTgX5CHDoJxagFpx9nGK0F2H0Y=";
+
+  nativeBuildInputs = [
+    helm
+    sops
+  ];
+
+  meta = with lib; {
+    description = "A Helm plugin to encrypt secrets with SOPS";
```

```suggestion
    description = "Helm plugin to encrypt secrets with SOPS";
```
Not start with the definite or an indefinite article.